src/modules/class-wpml-elementor-module-with-items.php

```diff
@@ -22,8 +22,10 @@ abstract class WPML_Elementor_Module_With_Items implements IWPML_Page_Builders_M
 	 */
 	abstract protected function get_editor_type( $field );
 
-	/** @return string */
-	abstract protected function get_items_field();
+	/**
+	 * @return string
+	 */
+	abstract public function get_items_field();
```

Comment:
@dgwatkins can you clarify this line? Where is this method implemented and used?

Reply:
@andreasciamanna it's used in src/class-wpml-elementor-translatable-nodes.php:110 
`$element[ self::SETTINGS_FIELD ][ $node->get_items_field() ][ $item['index'] ] = $item;`

Reply:
@andreasciamanna, this abstract method was wrong since the beginning. All the classes implementing this interface are declaring a public method. I suggest @amElnagdy to fix it, but it has nothing to do with the purpose of this PR, so he added it as a separate commit.